### ISSUE TITLE
BA10E Construct a Profile HMM

### ISSUE THREAD

**Issue by weka511** [weka511/bioinformatics#83]

**Comment by weka511** [comment]
See answers to questions from Robert Goldberg-Alberts

1. A critical part of the problem not covered in the sample, additional data set, and textbook examples is having the first column of the input data set deleted. That involves I0, and counting the insertions there. My code does this as a separate case from the M,D, and I sets that follow thereafter. 
2. The theta in the textbook is NOT the theta used by the grading software. The Grading software uses 1.0- theta for theta. The grader uses the transformed value in the same manner so merely transforming theta = 1-theta produced the desired answers. [I have tried this and it still doesn't pass]
3. I rounded all decimals to three places and the grader accepted the answers. It may be that the input did NOT product numbers with more than 3 places (e.g. 0.25) for Andrey. I wonder whether we do this for 0 1nd 1?
